app/org/sagebionetworks/bridge/models/schedules/ScheduleStrategy.java

```diff
@@ -15,7 +13,8 @@
 @JsonTypeInfo( use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
 @JsonSubTypes({
     @Type(name="SimpleScheduleStrategy", value=SimpleScheduleStrategy.class),
-    @Type(name="ABTestScheduleStrategy", value=ABTestScheduleStrategy.class)
+    @Type(name="ABTestScheduleStrategy", value=ABTestScheduleStrategy.class),
+    @Type(name="CriteriaScheduleStrategy", value=CriteriaScheduleStrategy.class)
 })
```

Comment:
How do these JSON annotations work? Do we have any tests to verify they work as expected?